cookbooks/app/recipes/create.rb

```diff
@@ -65,6 +65,8 @@ def determine_adapter
   # the recipes are monit, nginx, and depending on the stack passenger5, puma, or unicorn
   app.recipes.each do |recipe|
     next if recipe == "memcached"
+    #skipping node::tcp recipe run. This acts as a workaround for nodejs apps till EYPP-11098 is fixed
+    next if recipe == "node::tcp"
```

Comment:
Do we need this?

Reply:
Aside from this, LGTM

Reply:
It's going to break NodeJS app on V6 in the future till EYPP-11098 is fixed :(